### PR TITLE
add more tiles

### PR DESCRIPTION
**Pull request by PalmerAL**

### CODE REVIEW

File: src/browser/theme.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ define((require, exports, module) => {
     'daringfireball.net': ['#fff', '#4a525a', IS_DARK],
     'firewatchgame.com': ['#EF4338', '#2D102B', IS_DARK],
     'whatliesbelow.com': ['#fff', '#74888B', IS_DARK],
-    'supertimeforce.com': ['#2EBCEC', '#051224', IS_DARK]
+    'supertimeforce.com': ['#2EBCEC', '#051224', IS_DARK],
+    'huffingtonpost.com': ['#fff', '#2E7360', IS_DARK]
```
Comment (gordonbrander) [inline src/browser/theme.js]: I don't think we want to give this a dark background. The goal is to blend in with the page and draw the eye away from controls.

**Comment by gordonbrander** [comment]
BBC tile is off-center

**Comment by PalmerAL** [comment]
fixed.

**Comment by gordonbrander** [comment]
@PalmerAL hey, thanks for your work on this. I'm sorry to say that BBC is still off-center. Also, hadn't noticed this before, but Bing is as well.

Added some measurements to show what I mean. Photoshop or any other image manipulation tool should be able to take care of this for you via the `align` tool. Also just realized we'll want to make sure these are exported for web. Photoshop `File -> Save for web`. https://imageoptim.com/ is a great free tool for taking care of exporting for web if you don't have Photoshop.

screen shot 2015-03-13 at 11 00 54
screen shot 2015-03-13 at 10 59 46

Thanks!

**Comment by PalmerAL** [comment]
Okay, I think the tiles should be centered and optimized now. Please let me know if any further changes are needed.

**Comment by gordonbrander** [comment]
Looks good. Thanks!

**Pull request merged**